devicon.json

```
@@ -6383,6 +6383,29 @@
         "color": "#4878be",
         "aliases": []
     },
+    {
+        "name": "notion",
+        "altnames": [],
+        "tags": [
+            "project-management"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "plain"
+            ],
+            "font": [
+                "original"
```

The plain icon should be used as a font instead of the original, since original has multiple colors and paths.
```suggestion
                "plain"
```